api.js

```diff
@@ -211,10 +245,24 @@ exports.init = function (sbot, config) {
   }
 
   return {
-    getRoot,
-    findOrCreate,
-    findAndTombstone,
-    findById,
     branchStream,
+    findOrCreate: v1FindOrCreate,
+
+    advanced: {
+      getRoot,
+      findOrCreate,
+      findAndTombstone,
+      findById,
+
+      // TODO add API for whether to encrypt the announce message for a subfeed
+      // for sympathetic replication?
+      //
+      // ssb.db.create // allows
+      //
+      // QUESTION: should the groupKey be used for the determination of the location of the group subfeed (ideally don't use the same key for two different things right?)
```

Comment:
TODO - update spec to pin this down

Reply:
By the way, I don't think TODOs should be inside code, we should aim for converting these to issues before we merge this PR.